eng/pipelines/handlers.yml

```diff
@@ -228,9 +228,11 @@ stages:
                   ArtifactName: ${{ BuildPlatform.artifact }}
 
   - template: common/security_compliance.yml
-  - template: common/localization-handoff.yml                     # Process outgoing strings [Localization Handoff]
-  - template: common/localization-handback.yml                    # Process incoming translations and Create PR to main [Localization Handback]
-  - template: common/merge-translations-update.yml                # Validating incoming translations strings and merge PR [Localization Handback]
+
+  - ${{ if eq(variables['System.TeamProject'], 'devdiv') }}:
```

Comment:
i don't think we can do this, since these will need to run for PR's that the system opens, since PR's only run on public it will fail to run the last step to validate translations.

Reply:
Going to merge for now, but will need to test , but my gut tells me we need this on prs